cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/store/DefaultStoreTest.java

```diff
@@ -700,8 +700,9 @@ public void testHistoryDeletion() throws Exception {
     store.setStart(mapreduceProgramId1, "mrRun1", now - 1000);
     store.setStop(mapreduceProgramId1, "mrRun1", now, ProgramController.State.COMPLETED.getRunStatus());
 
-    store.setStart(workflowProgramId1, "wfRun1", now - 1000);
-    store.setStop(workflowProgramId1, "wfRun1", now, ProgramController.State.COMPLETED.getRunStatus());
+    RunId runId = RunIds.generate(System.currentTimeMillis());
+    store.setStart(workflowProgramId1, runId.getId(), now - 1000);
```

Comment:
though it's not affecting the test case, since the startTime is now-1000, shouldn't we use 
``RunIds.generate(System.currentTimeMillis())`` as runid ?